spec.html

```diff
@@ -30736,6 +30749,10 @@ <h2>Syntax</h2>
           CharacterEscape[?U]
       </emu-grammar>
 
+      <emu-note>
+	      <p>The syntax is modified and extended in <emu-xref href="#sec-regular-expressions-patterns"></emu-xref>.</p>
```

Comment:
I'd use wording more like this:

```suggestion
	      <p>A number of productions in this section are given alternative definitions in section <emu-xref href="#sec-regular-expressions-patterns"></emu-xref>.</p>
```